src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 71 to help you plan.
+
+{# wf_updated_on: 2018-11-26 #}
+{# wf_published_on: 2018-11-13 #}
```

Comment:
be sure to update these dates as we get closer to merging.

Reply:
Done.